plugins/prometheus/info.go

```diff
@@ -26,19 +26,19 @@ func registerInfoMetrics() {
 	}
 	infoApp.WithLabelValues(banner.AppName, banner.AppVersion, nodeID).Set(1)
 
-	sync = prometheus.NewGauge(prometheus.GaugeOpts{
-		Name: "sync",
+	syncStatus = prometheus.NewGauge(prometheus.GaugeOpts{
+		Name: "syncStatus",
```

Comment:
```suggestion
		Name: "sync",
```
the name of the gauge is important!